test/Vagrantfile

```diff
@@ -32,6 +31,7 @@ Vagrant.configure("2") do |config|
         server.vm.provision "file", source: "provision", destination: "/tmp/provision"
         server.vm.provision "shell" do |sh|
             sh.path = "./provision/runtime_install.sh"
+            sh.env = {}
```

Comment:
Not sure if it is meaningful to have empty env like this.

Reply:
For now CILIUM_USE_ENVOY=1 needs to be defined in the runtime environment of cilium-agent. It seems to me that this is the place where it needs to be defined.